static/js/lib/url.js

```diff
@@ -5,3 +5,5 @@ export const channelURL = (channelName: string) => `/channel/${channelName}`
 export const postDetailURL = (channelName: string, postID: string) => `/channel/${channelName}/${postID}`
 
 export const newPostURL = (channelName: string) => `/create_post/${channelName}`
```

Comment:
`create_post` was chosen because `/channel/${channelName}/posts` would have collided with the `${postID}`, which was already a URL when I started working on it. I'm not opposed to rethinking the URLs but it is outside the scope here. It's not a `POST` request because `/create_post/${channelName}` is for the page that shows the form to create the post, not the endpoint for creating the post itself. That is at `/api/v0/channels/{channelName}/posts/`